components/bl-mapbox-component/src/helpers.js

```diff
@@ -96,12 +96,19 @@ export const initMapboxLibrary = (mapRef, mapContainerRef, component, eventHandl
   createActions(mapRef, component);
 
   if (directions) {
-    map.addControl(
-      new MapboxDirections({
-        accessToken: mapboxgl.accessToken,
-      }),
-      'top-left'
-    );
+    const directions = new MapboxDirections({
+      accessToken: mapboxgl.accessToken,
```

Comment:
Could we get all needed in this function parts of mapboxgl using destruction?

`const {Map, accessToken, FullscreenControl ... } = Mapbox`


Also change it

`import mapboxgl from './lib/mapbox';` => `import Mapbox from './lib/mapbox';`